vagrant/Vagrantfile

```diff
@@ -799,7 +799,7 @@ Vagrant.configure('2') do |config|
 
       c.vm.provision 'install-lustre-client',
                      type: 'shell',
-                     run: 'never',
+                     run: 'once',
```

Comment:
This should stay as never, We shouldn't assume we always want to do this.

Reply:
But it is a client, we could even mount without this, no?

Reply:
Nevermind :)